Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml

```
@@ -20,6 +20,11 @@ configuration:
   name: Verify SSL
   required: false
   type: 8
+- display: Use system proxy settings
+  name: system_proxy
```

use our standard naming
```suggestion
  name: proxy
```